packages/blockchain-wallet-v4-frontend/src/scenes/Home/PriceChart/Footer/index.js

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { FormattedMessage } from 'react-intl'
+import { LinkContainer } from 'react-router-bootstrap'
+
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+import { actions, selectors } from 'data'
+import { getState } from './selectors'
+import { Button } from 'blockchain-info-components'
+
+const Wrapper = styled.div`
+  display: flex;
+  justify-content: space-around;
+  z-index: 2;
+  margin-bottom: 37px;
+  width: 100%;
+`
+
+const BuyTradeButton = styled(Button)`
+  max-width: 200px;
+  width: 100%;
+
+  &:first-child {
+    margin: 0 4px;
+  }
+
+  &:last-child {
+    margin-right: 4px;
+  }
+`
+
+const Footer = ({ actions, coinifySupported, priceChart }) => {
+  const [coinName, setCoinName] = useState('Bitcoin')
+
+  useEffect(() => {
+    actions.checkCountryStateSupported()
+  })
+
+  useEffect(() => {
+    switch (priceChart.coin) {
+      case 'BTC':
+        return setCoinName('Bitcoin')
+      case 'ETH':
+        return setCoinName('Ethereum')
+      case 'BCH':
+        return setCoinName('Bitcoin Cash')
+      case 'XLM':
+        return setCoinName('Stellar')
+      default:
+        setCoinName('Bitcoin')
```

Comment:
Should not hardcode coin names, they are all set in wallet-options. Pass wallet options into this component and use that instead.